### PR TITLE
[DOCS-1407]new homepage for docs, including design commons design patterns and j…

### PR DESCRIPTION
**Pull request by HugoJLFerreira**
…est testing

Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

  
  sample:
  
image

### CODE REVIEW

**Review by doucol — APPROVED** [review]

**Comment by doucol** [comment]
@HugoJLFerreira  - Code review is fine. Just a few questions / issues:
- I think we may need to get some updated images with transparent background
- Some of the spacing on the home page - for example: should we left-justify the h1/h2 on that page?
- Seems like there is some other spacing that feels just slightly off - but maybe I just need to warm up to it :) 
@ctauchen

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+      ],
+      linkDescription: 'Learn more about Calico Open Source',
+      image: 'img/calico-open-source.svg',
+    },
+    {
+      title: 'Calico Enterprise',
+      link: '/calico-enterprise/latest/about-calico-enterprise',
+&nbsp;
+      description: [
+        'Includes Calico Open Source core networking and network policy, but adds advanced features for networking, network policy, visibility and troubleshooting, threat defense, and compliance reports. ',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: - A paid, self-managed security and observability solution for containers and Kubernetes. 
- Extends Calico Open Source’s networking and network security capabilities to offer more advanced security and observability capabilities for organizations running Kubernetes at scale.

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+      ],
+      linkDescription: 'Learn more about Enterprise',
+      image: 'img/calico-enterprise.drawio.svg',
+    },
+    {
+      title: 'Calico Cloud',
+&nbsp;
+      link: 'calico-cloud',
+      description: [
+        'The SaaS version of Calico Enterprise. It adds Image Assurance to scan and detect vulnerabilities in images, and container threat defense to detect malware. It also adds onboarding tutorials, and eliminates the cost to manage Elasticsearch logs and storage that comes with Calico Enterprise.',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: - A pay-as-you-go, SaaS application that provides comprehensive container security across the entire container lifecycle (build, deploy, runtime).
- A fully-managed version of Calico Enterprise that adds container image scanning and advanced threat detection capabilities.

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">/* eslint-disable max-len */
+&nbsp;
+export default {
+  aboutInfo: [
+    {
+      title: 'Calico Open Source',
+      link: '/calico/latest/about/',
+&nbsp;
+      description: [
+        'The base product that comprises both Calico Enterprise and Calico Cloud. It provides the core networking and network policy features.',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: - A free, self-managed networking and security solution for containers, virtual machines, and native host-based workloads.
- Provides networking, network policy, and IP address management capabilities for cloud-native applications.

File: coverage/lcov-report/data/exploreInfo.ts.html [hunk 4]
```diff
@@ -0,0 +1,178 @@
+
```
Comment (bmckercher123) [inline coverage/lcov-report/data/exploreInfo.ts.html]: Delete this card (Explore the basics) entirely. Text has been moved to "About Tigera products"

File: coverage/lcov-report/components/SelectDocs/index.tsx.html [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+  &lt;Flex
+    sx={sectionOuterStyles(isDarkMode)}
+    {...rest}
+  &gt;
+    &lt;Heading
+      as='h2'
+      size='md'
+      sx={{ ...heading2Styles(isDarkMode), ...headerTextStyle }}
+    &gt;
+      Select your doc set
```
Comment (bmckercher123) [inline coverage/lcov-report/components/SelectDocs/index.tsx.html]:
```suggestion
      Select a doc set
```

File: coverage/lcov-report/components/Explore/index.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
+
```
Comment (bmckercher123) [inline coverage/lcov-report/components/Explore/index.html]: Delete this section, "Explore the basic". No longer needed.

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+      image: 'img/calico-open-source.svg',
+    },
+    {
+      title: 'Calico Enterprise',
+      link: '/calico-enterprise/latest/about-calico-enterprise',
+&nbsp;
+      description: [
+        'Includes Calico Open Source core networking and network policy, but adds advanced features for networking, network policy, visibility and troubleshooting, threat defense, and compliance reports. ',
+      ],
+      linkDescription: 'Learn more about Enterprise',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: Go to docs

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+      image: 'img/calico-enterprise.drawio.svg',
+    },
+    {
+      title: 'Calico Cloud',
+&nbsp;
+      link: 'calico-cloud',
+      description: [
+        'The SaaS version of Calico Enterprise. It adds Image Assurance to scan and detect vulnerabilities in images, and container threat defense to detect malware. It also adds onboarding tutorials, and eliminates the cost to manage Elasticsearch logs and storage that comes with Calico Enterprise.',
+      ],
+      linkDescription: 'Learn more about Calico Cloud',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: Go to docs

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 9]
```diff
@@ -0,0 +1,322 @@
...
+      image: 'img/calico-cloud-features.svg',
+    },
+  ],
+  bestFit: [
+    {
+      title: 'Calico Open Source',
+      costSupport: ['Free, self-managed'],
+      target: 'Users',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: target: '**Users** who want best-in-class networking and network policy capabilities.'

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 10]
```diff
@@ -0,0 +1,322 @@
...
+      bestFit: [' who want best-in-class networking and network policy capabilities.'],
+    },
+    {
+      title: 'Calico Enterprise',
+      costSupport: ['Paid, self-managed'],
+      image: '/img/calico-cloud-logo.webp',
+&nbsp;
+      target: 'Enterprise teams',
+&nbsp;
+      bestFit: [
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: ' - Who need full control to customize their networking security deployment to meet regulatory and',
        ' compliance requirements for Kubernetes at scale.',
        ' - Who want Tigera Customer Support best practices for day zero to production best practices',
        ' custom training and workshops, and Solution Architects to customize solutions.',

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 11]
```diff
@@ -0,0 +1,322 @@
...
+        ' custom training and workshops, and Solution Architects to customize solutions.',
+      ],
+    },
+    {
+      title: 'Calico Cloud',
+      costSupport: ['Pay-as-you-go, SaaS'],
+      image: '/img/calico-enterprise-logo.webp',
+&nbsp;
+      target: 'Small teams',
+      bestFit: [
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: ' - Who need to manage the full spectrum of compliance in a web-based console for novice users:',
        '- Secure clusters, pods, and applications',
        '- Scan images for vulnerabilities',
        '- Web-based UI for visibility to troubleshoot Kubernetes',
        '- Detect and mitigate threats',
        '- Run compliance reports',

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 12]
```diff
@@ -0,0 +1,322 @@
...
+        ' who need to manage the full spectrum of compliance in a web-based console for novice users:',
+        '- Secure clusters, pods, and applications',
+        '- Scan images for vulnerabilities',
+        '- Web-based UI for visibility to troubleshoot Kubernetes',
+        '- Detect and mitigate threats',
+        '- Run compliance reports',
+      ],
+      target2: 'Enterprise teams',
+      bestFit2: [
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: Add a line above this section...and add bullet

' - Who want to scale their Calico Enterprise on-premises deployments by providing more self-service to developers.',

File: coverage/lcov-report/data/compareInfo.ts.html [hunk 13]
```diff
@@ -0,0 +1,322 @@
...
+        ' who need full control to customize their networking security deployment to meet regulatory and',
+        ' compliance requirements for Kubernetes at scale.',
+        ' Teams who require Tigera Customer Support for best practices to get from day zero to production,',
+        ' custom training and workshops, and Solution Architects to customize solutions.',
+      ],
+    },
+    {
+      title: 'Calico Cloud',
+      costSupport: ['Pay-as-you-go, SaaS'],
+      image: '/img/calico-enterprise-logo.webp',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/compareInfo.ts.html]: Center text under logo to match others.

File: src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap [hunk 14]
```diff
@@ -0,0 +1,3158 @@
...
+            </tr>
+            <tr
+              class="css-0"
+              role="row"
+            >
+              <td
+                class="css-1l7w1il"
+                role="gridcell"
+              >
+                Policy recommendation
```
Comment (bmckercher123) [inline src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap]: Change to plural...

Policy recommendations

with checkmarks for Calico Enterprise and Calico Cloud

File: src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap [hunk 15]
```diff
@@ -0,0 +1,3158 @@
...
+            </tr>
+            <tr
+              class="css-0"
+              role="row"
+            >
+              <td
+                class="css-1l7w1il"
+                role="gridcell"
+              >
+                Container threat detection
```
Comment (bmckercher123) [inline src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap:2634]: Container threat detection should be true only for Calico Cloud.

File: src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap [hunk 16]
```diff
@@ -0,0 +1,3158 @@
...
+            </tr>
+            <tr
+              class="css-0"
+              role="row"
+            >
+              <td
+                class="css-1l7w1il"
+                role="gridcell"
+              >
+                Egress gateway
```
Comment (bmckercher123) [inline src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap:312]: Calico OSS should be false for egress gateway

File: src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap [hunk 17]
```diff
@@ -0,0 +1,3158 @@
...
+            </tr>
+            <tr
+              class="css-0"
+              role="row"
+            >
+              <td
+                class="css-1l7w1il"
+                role="gridcell"
+              >
+                Onboarding tutorials and lab cluster
```
Comment (bmckercher123) [inline src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap:914]: True only for Calico Cloud.

File: src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap [hunk 18]
```diff
@@ -0,0 +1,3158 @@
...
+                </svg>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <h3
+        class="chakra-heading css-1uy7oy0"
+      >
+        Non-cluster hosts
```
Comment (bmckercher123) [inline src/___new___/components/FeatureTableComparison/__test__/__snapshots__/index.test.tsx.snap:1416]: Recently changed...

"Restrict traffic.." and "automatic host endpoints" for Calico Cloud should be false.

File: src/pages/innerLayout.js [hunk 19]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Explore from '../___new___/components/Explore';
```
Comment (bmckercher123) [inline src/pages/innerLayout.js]: Remove Explore throughout

File: coverage/lcov-report/components/LearnMoreButton/index.tsx.html [hunk 20]
```diff
@@ -0,0 +1,175 @@
...
+      as={Link}
+      variant='ghost'
+      size='lg'
+      {...{
+        href,
+        'aria-label': ariaLabel,
+      }}
+      {...rest}
+    &gt;
+      Go to docs
```
Comment (bmckercher123) [inline coverage/lcov-report/components/LearnMoreButton/index.tsx.html]: Can you align left to match preceding paragraph?

File: src/___new___/data/compareInfo.ts [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+    },
+    {
+      title: 'Calico Cloud',
+      costSupport: ['Pay-as-you-go, SaaS'],
+      image: '/img/calico-enterprise-logo.webp',
+
+      target: 'Small teams',
+      bestFit: [
+        ' - Who need to manage the full spectrum of compliance in a web-based console for novice users:',
+        '- Secure clusters, pods, and applications',
```
Comment (bmckercher123) [inline src/___new___/data/compareInfo.ts]: These bullets (starting with - Secure clusters, pods, and applications) are items under the lead bullet so need to be indented.

File: src/___new___/data/featureTableComparisonInfo.ts [hunk 22]
```diff
@@ -0,0 +1,349 @@
...
+          CalicoEntreprise: '',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Observability and troubleshooting',
+      content: [
+        {
+          rowHeader: 'Application level observability and troubleshooting',
```
Comment (bmckercher123) [inline src/___new___/data/featureTableComparisonInfo.ts]:
```suggestion
          rowHeader: 'Application-level observability and troubleshooting',
```

File: src/___new___/data/compareInfo.ts [hunk 23]
```diff
@@ -0,0 +1,82 @@
...
+      title: 'Calico Enterprise',
+      costSupport: ['Paid, self-managed'],
+      image: '/img/calico-cloud-logo.webp',
+
+      target: 'Enterprise teams',
+
+      bestFit: [
+        ' - Who need full control to customize their networking security deployment to meet regulatory and',
+        ' compliance requirements for Kubernetes at scale.',
+        ' - Who want Tigera Customer Support best practices for day zero to production best practices',
```
Comment (bmckercher123) [inline src/___new___/data/compareInfo.ts]:
```suggestion
        ' - Who want Tigera Customer Support for day-zero to production best practices',
```

File: src/___new___/data/featureTableComparisonInfo.ts [hunk 24]
```diff
@@ -0,0 +1,349 @@
...
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Service Graph',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Threat graph',
```
Comment (bmckercher123) [inline src/___new___/data/featureTableComparisonInfo.ts:243]: Do we have a threat graph? I'm not sure where I got this item. Is it for container threat defense?

File: src/___new___/data/featureTableComparisonInfo.ts [hunk 25]
```diff
@@ -0,0 +1,349 @@
...
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Multi-cluster management',
+      content: [
+        {
```
Comment (bmckercher123) [inline src/___new___/data/featureTableComparisonInfo.ts:271]: Add this item:

{
          rowHeader: 'Multi-cluster management',
          CalicoOpenSource: '',
          CalicoEntreprise: 'Y',
          CalicoCloud: '',
        },

File: src/___new___/data/featureTableComparisonInfo.ts [hunk 26]
```diff
@@ -0,0 +1,349 @@
...
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Workload-centric Web Application Firewall (WAF)',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Honeypods to see intruder activity',
```
Comment (bmckercher123) [inline src/___new___/data/featureTableComparisonInfo.ts]:
```suggestion
          rowHeader: 'Honeypods to view intruder activity',
```

File: coverage/lcov-report/components/LearnMoreButton/index.tsx.html (hunk 20 above)
Comment (bmckercher123) [inline coverage/lcov-report/components/LearnMoreButton/index.tsx.html]: Suggest that we change these links to:

      Calico Open Source documentation
      Calico Enterprise documentation
      Calico Cloud documentation

File: coverage/lcov-report/data/featureTableComparisonInfo.ts.html [hunk 27]
```diff
@@ -0,0 +1,1132 @@
...
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Service Graph',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Threat graph',
```
Comment (bmckercher123) [inline coverage/lcov-report/data/featureTableComparisonInfo.ts.html]: Remove "Threat graph"...this is just Marketing speak for "Service Graph"

File: src/___new___/data/featureTableComparisonInfo.ts (hunk 22 above)
Comment (ctauchen) [reply src/___new___/data/featureTableComparisonInfo.ts]: Fixed.

File: src/___new___/data/compareInfo.ts (hunk 23 above)
Comment (ctauchen) [reply src/___new___/data/compareInfo.ts]: Fixed.

File: src/___new___/data/featureTableComparisonInfo.ts (hunk 25 above)
Comment (ctauchen) [reply src/___new___/data/featureTableComparisonInfo.ts:271]: Do you mean to have a second item under 'Multi-cluster management' that is also called 'Multi-cluster management'?

File: src/___new___/data/featureTableComparisonInfo.ts (hunk 26 above)
Comment (ctauchen) [reply src/___new___/data/featureTableComparisonInfo.ts]: Fixed.

File: coverage/lcov-report/data/featureTableComparisonInfo.ts.html (hunk 27 above)
Comment (ctauchen) [reply coverage/lcov-report/data/featureTableComparisonInfo.ts.html]: Removed

File: src/___new___/data/compareInfo.ts (hunk 21 above)
Comment (HugoJLFerreira) [reply src/___new___/data/compareInfo.ts]: would something like this suit? 
image

Comment (ctauchen) [reply src/___new___/data/compareInfo.ts]: That can do in a pinch. Better would be to add a space and one hyphen? I tried adding spaces, but they never displayed in the build.

Comment (HugoJLFerreira) [reply src/___new___/data/compareInfo.ts]: fixed like so:
image

**Pull request merged**